### PR TITLE
Installation script for linux by @niebert

### PR DESCRIPTION
**Pull request by ann0see**
@trebmuh since this will go to the new knowledge base, I closed #97 and opened this one. No need to start with this now.

### CODE REVIEW

**Comment by ann0see** [comment]
Can we merge it or should we wait?

**Comment by ann0see** [comment]
@trebmuh can we merge this? I made some smaller changes.

File: _posts/2020-09-20-Linux-Install-Script.md [hunk 1]
```diff
@@ -6,11 +6,9 @@ author: "niebert"
 lang: "en"
 ---
 
-If you plan to be installing Jamulus on many Linux machines, you may want to try this script.
+IThanks to niebert, if you plan to install Jamulus on many Linux machines, you can try this script.
```
Comment (trebmuh) [inline _posts/2020-09-20-Linux-Install-Script.md]: Looks like a typo here.

**Review by trebmuh — CHANGES_REQUESTED** [review]

File: _posts/2020-09-20-Linux-Install-Script.md (hunk 1 above)
Comment (ann0see) [reply _posts/2020-09-20-Linux-Install-Script.md]: I already solved that.

Comment (trebmuh) [reply _posts/2020-09-20-Linux-Install-Script.md]: ok

Comment (ann0see) [reply _posts/2020-09-20-Linux-Install-Script.md]: You should be able to review it again (with a new GitHub review)

File: _posts/2020-09-20-Linux-Install-Script.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+<!--more-->
+
+The following example was tested on Linux Mint and combines all the commands to install Jamulus on Linux into one script (currently for Ubuntu/Linux Mint).
+
+## How it works
+
+To incorporate the different commands for different Linux distributions, variables define the distribution and the release for which the installation script should be performed. The following focuses on Ubuntu with release 18.04 as example. So the suggested script name for the release is e.g. `install4ubuntu18_4.sh`. The script commands are generic so that the installation could also be modified so that they work on other Linux distributions.
+
+### Installation dependent on Linux Distribution
+The following script calls different installation commands dependent on the Linux distribution.
```
Comment (trebmuh) [inline _posts/2020-09-20-Linux-Install-Script.md]: "dependent" -> "depending"

File: _posts/2020-09-20-Linux-Install-Script.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+The variable `DISTRO` defines which commands are executed. Set the variable dependent on the Linux distribution you are using.
```
Comment (trebmuh) [inline _posts/2020-09-20-Linux-Install-Script.md]: same, "dependent" -> "depending"

File: _posts/2020-09-20-Linux-Install-Script.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+* `DISTRO="Ubuntu"` for a Ubuntu or Linux Mint
+* `DISTRO="Debian"` for a Debian or Raspbian Linux
+* `DISTRO="Fedora"` for a Fedora Linux
+Furthermore if the installation is dependent of the release the variable `LINVERSION` is introduced but is currently not used. In the Ubuntu `if` statement there is an example how version dependent installation calls can be performed.
```
Comment (trebmuh) [inline _posts/2020-09-20-Linux-Install-Script.md]: one more, "dependent" -> "depending"

File: _posts/2020-09-20-Linux-Install-Script.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: "Linux Bash Installation Script"
+heading: "Bash Installation Script"
+author: "niebert"
+lang: "en"
+---
+
+Thanks to niebert, if you plan to install Jamulus on many Linux machines, you can try this script.
```
Comment (trebmuh) [inline _posts/2020-09-20-Linux-Install-Script.md]: What about a link for niebert here?
`[niebert](https://github.com/niebert)`

**Review by trebmuh — CHANGES_REQUESTED** [review]
A few suggestion.
Great overall rework!

File: _posts/2020-09-20-Linux-Install-Script.md (hunk 1 above)
Comment (trebmuh) [reply _posts/2020-09-20-Linux-Install-Script.md]: I was doing it :)

**Comment by ann0see** [comment]
>  Great overall rework!

I didn't do much here. ;-). 

You can merge it if you think it's ok.

File: _posts/2020-09-20-Linux-Install-Script.md (hunk 3 above)
Comment (ann0see) [reply _posts/2020-09-20-Linux-Install-Script.md]: fixed

File: _posts/2020-09-20-Linux-Install-Script.md (hunk 4 above)
Comment (ann0see) [reply _posts/2020-09-20-Linux-Install-Script.md]: done

File: _posts/2020-09-20-Linux-Install-Script.md (hunk 5 above)
Comment (ann0see) [reply _posts/2020-09-20-Linux-Install-Script.md]: added

**Comment by trebmuh** [comment]
Merged.

**Pull request merged**